pkg/services/pluginsintegration/angularpatternsstore/store_test.go

```diff
@@ -59,7 +60,7 @@ func TestAngularPatternsStore(t *testing.T) {
 		})
 
 		t.Run("invalid timestamp stored", func(t *testing.T) {
```

Comment:
nit: I guess you can remove these test since it's written now in the common implementation